Users/abehenderson/CSX/Callbacks/intersection.js

```diff
@@ -0,0 +1,50 @@
+function intersection(arrays) {
+  
+  let result = arrays.reduce((acc, array) => {
```

Comment:
You should always prefer `const` to `let` unless you are creating a value that _must_ be changed later. Using `const` prevents that from happening by accident.

Reply:
But for my money, I'd simply get rid of "let result =" and just "return arrays.reduce...", not forgetting to nix "return result" at the end of the function.

Reply:
Yep, agree and agree.  Thanks for the reminders.